src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.Designer.cs

```diff
@@ -9685,7 +9676,7 @@ internal class Strings {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Error code 0x{0}; state {1}.
+        ///   Looks up a localized string similar to Error code 0x{0}.
```

Comment:
Other changes except this one are unrelated in this file.

Reply:
Davoud, I have addressed this change.